Translation Docs Text Correction (#23)
TheCloudMasters/docs: data/articles/docs/files/4-metadata-types/labels-and-translations/labels-and-translations.md

# Labels and Translations

> This is a really handy feature if you are building apps for different countries or people of different languages. ues.io checks the user's language settingnand then applies the translated labels for that language in the application user interface. Currently there are 7 supported languages:

Reviewer: ```suggestion
This is a really handy feature if you are building apps for different countries or people of different languages. ues.io checks the user's language setting and then applies the translated labels for that language in the application user interface. Currently there are 7 supported languages:
```

> This is a really handy feature if you are building apps for different countries or people of different languages. ues.io checks the language setting of the user's local machine and then applies the language and the translated labels in the application user interface. Currently there are 7 supported languages:

Reviewer: ```suggestion
This is a really handy feature if you are building apps for different countries or people of different languages. ues.io checks the user's language settingnand then applies the translated labels for that language in the application user interface. Currently there are 7 supported languages:
```

-   There are 11 default labels such as Create, Delete, etc. that can be translated in your chosen language.

> -   If you need more labels for your Button actions or Field labels or elsewhere then you can create a new label and transalte it in your chosen translation language.

Reviewer: ```suggestion
-   If you need more labels for your Button actions or Field labels or elsewhere then you can create a new label and translate it in your chosen translation language.
```
